analyzers/tests/SonarAnalyzer.Test/Rules/UnnecessaryUsingsTest.cs

```diff
@@ -73,7 +73,7 @@ public class UnnecessaryUsingsTest
         [DataRow("RandomFile_Imports.cshtml")]
         [DataRow("_Imports.cshtml")]
         public void UnnecessaryUsings_RazorViewImportsSimilarCshtmlFile_IssuesReported(string fileName) =>
-            builder
+            builder.AddReferences(NuGetMetadataReference.SystemTextJson("7.0.4"))
```

Comment:
I would move the `AddReferences` after `AddSnippet` (same as before, also below)